app/src/main/java/com/asfoundation/wallet/ui/iab/MergedAppcoinsView.kt

```diff
@@ -0,0 +1,26 @@
+package com.asfoundation.wallet.ui.iab
+
+import io.reactivex.Observable
+
+interface MergedAppcoinsView {
+
+  fun showError(errorMessage: Int)
```

Comment:
we should use Android annotation such as @IdRes as ofter as possible

Reply:
*often

Reply:
*StringRes was used instead